Sources/tart/Commands/Run.swift

```diff
@@ -15,10 +15,13 @@ struct Run: AsyncParsableCommand {
   var name: String
 
   @Flag(help: ArgumentHelp(
-          "Don't open a UI window.",
+          "Don't open an UI window.",
```

Comment:
```suggestion
          "Don't open a UI window.",
```
Only need to use 'an' before a vowel sound, not a vowel letter https://www.thesaurus.com/e/grammar/a-vs-an/